cypress/integration_defunct/sales/open_picking_terminal_and_filter.js

```diff
@@ -0,0 +1,78 @@
+/** todo The test is done but is failing due to https://github.com/metasfresh/me03/issues/2381.
+ * After this issue will be fixed, the test will pass.
+ *  It will need to be moved from integration/defunct dir. */
+import { BPartner } from '../../support/utils/bpartner';
+import { DiscountSchema } from '../../support/utils/discountschema';
+import { Builder } from '../../support/utils/builder';
+import { humanReadableNow } from '../../support/utils/utils';
+import { SalesOrder, SalesOrderLine } from '../../support/utils/sales_order';
+import {
+  applyFilters,
+  selectNotFrequentFilterWidget,
+  selectFrequentFilterWidget,
+  toggleNotFrequentFilters,
+  toggleFrequentFilters,
+} from '../../support/functions';
+
+describe('Create Sales order', function() {
```

Comment:
Isn't the description a bit misleading? This test is doing way more. Should we name it the same as the file maybe?

Reply:
done.